qt/python/mantidqt/widgets/src/_widgetscore.sip

```diff
@@ -0,0 +1,98 @@
+%ModuleCode
+#include "MantidQtWidgets/Common/Message.h"
```

Comment:
I don't think we need the following 2 includes at this level - just `#include "MantidQtWidgets/Common/Message.h"`. The main reason is to allow the `using namespace MantidQt::MantidWidgets`. Can we move the classes in `MantidQt::API` to `MantidQt::MantidWidgets`? I'd likely to slowly kill of the `MantidQt::API` namespace as we don't have the library anymore.